common/djangoapps/third_party_auth/migrations/0004_lti_tool_consumers.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=C,E,F,R,W
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding model 'LTIProviderConfig'
+        db.create_table('third_party_auth_ltiproviderconfig', (
+            ('id', self.gf('django.db.models.fields.AutoField')(primary_key=True)),
+            ('change_date', self.gf('django.db.models.fields.DateTimeField')(auto_now_add=True, blank=True)),
+            ('changed_by', self.gf('django.db.models.fields.related.ForeignKey')(to=orm['auth.User'], null=True, on_delete=models.PROTECT)),
+            ('enabled', self.gf('django.db.models.fields.BooleanField')(default=False)),
+            ('icon_class', self.gf('django.db.models.fields.CharField')(default='fa-signin', max_length=50)),
+            ('name', self.gf('django.db.models.fields.CharField')(max_length=50)),
+            ('lti_consumer_key', self.gf('django.db.models.fields.CharField')(max_length=255)),
+            ('lti_consumer_secret', self.gf('django.db.models.fields.CharField')(max_length=255)),
+            ('lti_max_timestamp_age', self.gf('django.db.models.fields.IntegerField')(default=10)),
```

Comment:
I'm not sure why, but this migration is not correct - it's missing the `secondary`, `skip_registration_form`, and `skip_email_verification` fields inherited from `ProviderConfig`. When I try testing this branch, I get:

```
DatabaseError at /admin/third_party_auth/ltiproviderconfig/
(1054, "Unknown column 'third_party_auth_ltiproviderconfig.secondary' in 'field list'")
```

Which is odd, since the "frozen" models generated below for other ProviderConfig descendants do include the inherited fields, so the code was presumably there are the time the migration was generated.